docs/quick_start.md

```diff
@@ -45,35 +46,40 @@ libtorch model which calculates the sum and difference of two inputs. Run the
 Model Analyzer `profile` subcommand inside the container with:
 
 ```
-$ model-analyzer profile -m /quick_start_repository/ \
-    --profile-models add_sub \
-    --run-config-search-max-concurrency 2 \
-    --run-config-search-max-instance-count 2 \
-    --run-config-search-preferred-batch-size-disable true
+$ model-analyzer profile -m . --profile-models add_sub
 ```
 
 If you already ran this earlier in the container, you can use the `--override-output-model-repository` option to overwrite the earlier results.
 
 This will perform a search across various config parameters on the `add_sub`
-model. This should take no more than a few minutes to finish. Here is some sample output:
+model. This will take around 60 minutes to finish. If you want a shorter run (1-2 minutes) for example purposes, you can run with the below additional options. Note that these options are not intended to find the best configuration:
 
 ```
+--run-config-search-max-concurrency 2 \
+--run-config-search-max-instance-count 2 \
+--run-config-search-preferred-batch-size-disable true
+```
+
```

Comment:
You should state here what those parameters do and how may configs will be tested.

Reply:
added 👍